src/pages/Login.js

```diff
@@ -44,6 +45,7 @@ export const Login = (props) => {
       dispatch({
         type: 'AUTHENTICATE'
       })
+      toast.success(`Logged in successfully as ${payload.user.username}`)
       navigate('/', { replace: true })
     } catch (e) {}
```

Comment:
do we need to handle this?

Reply:
We are calling the `login` service there. So, we need to handle any unexpected error in the the service. Although, there is already a `catch` block in the services itself to handle the errors. Should I remove this try-catch block?